tests/test_user_landing_page.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from django.shortcuts import reverse
+
+
+@pytest.mark.parametrize("dues_amount", (3, 100))
```

Comment:
parametrize -> parameterize?

Reply:
yep -- rushed this one out and missed that.  Can you try step number 6 on [this page](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request) to try and make an inline edit with this change?